plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestDefaultQueryBuilder.java

```diff
@@ -261,7 +261,7 @@ public void testBuildSqlWithDomainComplement()
     {
         TupleDomain<ColumnHandle> tupleDomain = TupleDomain.withColumnDomains(ImmutableMap.<ColumnHandle, Domain>builder()
                 // complement of a Domain with null not allowed
-                .put(columns.get(0), Domain.create(ValueSet.of(BIGINT, 128L, 180L, 233L), false).complement())
+                .put(columns.get(0), Domain.create(ValueSet.of(BIGINT, 128L, 180L, 233L, 234L, 1012L), false).complement())
```

Comment:
I think this test case was fine (it wasn't power of 2 anyway).

however, to fully test the change, we also need to assert on the parameters being bound.

Reply:
I've added a separate test case